From 106dd07846e1c364635551646343760c5c6430ee Mon Sep 17 00:00:00 2001
From: khm <khm@users.noreply.github.com>
Date: Wed, 6 Sep 2023 01:59:19 -0700
Subject: [PATCH] Update 60-input-id.hwdb: add TEX Shinobi (#29068)

* Update 60-input-id.hwdb: add TEX Shinobi

The TEX Shinobi keyboard with trackpoint incorrectly identifies as a mouse instead of a pointing stick.   This corrects it as suggested at https://gitlab.freedesktop.org/libinput/libinput/-/issues/932#note_2069967

Following the example of the Lite-On keyboard entry, this modalias specifies the mouse unit without tagging the device's other entries.

(cherry picked from commit c00c3d93d5970408a939dd1f6087180fe723716b)

Related: RHEL-5950
---
 hwdb.d/60-input-id.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-input-id.hwdb b/hwdb.d/60-input-id.hwdb
index 802ed9a1e7..1c84777fe0 100644
--- a/hwdb.d/60-input-id.hwdb
+++ b/hwdb.d/60-input-id.hwdb
@@ -75,3 +75,7 @@ id-input:modalias:input:b0003v046Dp4066e0111*
 id-input:modalias:input:b0003v068Ep00F2e0100*
  ID_INPUT_ACCELEROMETER=0
  ID_INPUT_JOYSTICK=1
+
+# TEX Shinobi Trackpoint
+id-input:modalias:input:b0003v04D9p0407e0111-e0,1,2,4*
+ ID_INPUT_POINTINGSTICK=1
